<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1730</width>
    <height>1217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1681</width>
      <height>1161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Grid Height</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Grid Width</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QSpinBox" name="setGridWidth">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Starting Cells</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="ResetButton">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Speed</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="StopButton">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="StartButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="setStartingCells">
       <property name="maximum">
        <number>6000</number>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QSpinBox" name="setUpdateSpeed">
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="setGridHeight">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QComboBox" name="Structure_comboBox">
       <item>
        <property name="text">
         <string>GosperGliderGun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SimkinGliderGun</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1730</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
